cookie_consent/views.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 from django.core.exceptions import SuspiciousOperation
-from django.contrib.auth.views import SuccessURLAllowedHostsMixin
+from django.contrib.auth.views import RedirectURLMixin
```

Comment:
This breaks on older versions, and as long as Django 3.2 is not end-of-life, it should still work on the supported versions.

The `RedirectURLMixin` does not exist there (see https://github.com/django/django/blob/stable/3.2.x/django/contrib/auth/views.py)

A better approach is to do a try-except import in the [`compat` module](https://github.com/jazzband/django-cookie-consent/blob/master/cookie_consent/compat.py) and import this mixin from there. You can then use the new one if it exists and fall back to `SuccessURLAllowedHostsMixin` for older versions.

Reply:
Thanks, and apologize for the mistake. This is my fist contribution :).

I have added the import to the compact module and made changes to the view. I hope it is correct now.

Reply:
thank you for your first contribution! And you don't need to apologize for that - everyone has their own preferred style and constraints, it's hard to take that all into account.